src/foam/nanos/notification/email/services.jrl

```diff
@@ -1,5 +1,5 @@
 p({"class":"foam.nanos.boot.NSpec", "name":"emailTemplateDAO",                 "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.PMDAO(x, new foam.dao.PipelinePMDAO(x, new foam.dao.SequenceNumberDAO(new foam.dao.java.JDAO(x, foam.nanos.notification.email.EmailTemplate.getOwnClassInfo(), \"emailTemplates\"))));","client":"{\"of\":\"foam.nanos.notification.email.EmailTemplate\"}" })
 p({"class":"foam.nanos.boot.NSpec", "name":"email",                            "lazy":true,   "serve":false, "authenticate":true, "serviceScript":"import foam.nanos.notification.email.AsyncEmailService; \nimport foam.nanos.notification.email.EmailService;\nimport foam.nanos.notification.email.NullEmailService; \nimport foam.nanos.notification.email.ProxyEmailService; \n \n ProxyEmailService pet = new AsyncEmailService.Builder(x).build(); \n EmailService es = new NullEmailService.Builder(x).build(); \npet.setDelegate(es);\n return pet;"})
-p({"class":"foam.nanos.boot.NSpec", "name":"emailMessageDAO",                  "lazy":true,   "serve":true, "serviceScript":"return new foam.dao.AuthenticatedDAO(\"emailMessage\", \n new foam.nanos.auth.CreatedAwareDAO.Builder(x).setDelegate(\n new foam.nanos.auth.CreatedByAwareDAO.Builder(x).setDelegate(\nnew foam.dao.SequenceNumberDAO(\n new foam.dao.FixedSizeDAO.Builder(x).setDelegate(\nnew foam.nanos.notification.email.EmailServiceDAO.Builder(x).setDelegate(\nnew foam.dao.WriteOnlyJDAO(x, \nfoam.nanos.notification.email.EmailMessage.getOwnClassInfo(), \"emails\"))\n.build()).build())).build()).build());", "client":"{\"of\":\"foam.nanos.notification.email.EmailMessage\"}"})
+p({"class":"foam.nanos.boot.NSpec", "name":"emailMessageDAO",                  "lazy":true,   "serve":true, "serviceScript":"return new foam.dao.AuthenticatedDAO(\"emailMessage\", \n new foam.nanos.auth.CreatedAwareDAO.Builder(x).setDelegate(\n new foam.nanos.auth.CreatedByAwareDAO.Builder(x).setDelegate(\nnew foam.dao.SequenceNumberDAO(\n new foam.dao.FixedSizeDAO.Builder(x).setDelegate(\nnew foam.dao.WriteOnlyJDAO(x, \nfoam.nanos.notification.email.EmailMessage.getOwnClassInfo(), \"emails\"))\n.build())).build()).build());", "client":"{\"of\":\"foam.nanos.notification.email.EmailMessage\"}"})
```

Comment:
Can we convert this to EasyDAO? 
name: localEmailMessageDAO serve:false serviceScript:
new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.email.EmailMessage.getOwnClassInfo()).setAuthenticate(false).setPermissionPrefix(\"emailMessage\").setCreatedAware(true).setCreatedByAware(true).setSeqNo(true).setInnerDAO(new foam.dao.FixedSizeDAO.Builder(x).setDelegate(new foam.dao.WriteOnlyJDAO(x, foam.nanos.notification.email.EmailMessage.getOwnClassInfo(), \"emails\")).build()).build()
with serve:false

There is a good chance this is already done on development. 
then also create an emailMessageDAO via EasyDAO, but all you need to set is the   setDecorator( x.get("localEmailMessageDAO")) .   this one is serve:true.